data/data-dictionary.csv

```diff
@@ -226,19 +494,44 @@ provider_type,Filter,Special college,,,,api-only,Establishment characteristics
+relative_access,Filter,All smaller rural settlements,rurality,Smaller rural settlement,,api-only,Personal characteristics
+relative_access,Filter,All unknown,rurality,Unknown,,api-only,Personal characteristics
+relative_access,Filter,All urban,rurality,Urban,,api-only,Personal characteristics
+relative_access,Filter,Further from a major town or city,rurality,Larger rural settlement,,api-only,Personal characteristics
+relative_access,Filter,Further from a major town or city,rurality,Smaller rural settlement,,api-only,Personal characteristics
+relative_access,Filter,Further from a major town or city,rurality,Urban,,api-only,Personal characteristics
+relative_access,Filter,Nearer to a major town or city,rurality,Larger rural settlement,,api-only,Personal characteristics
+relative_access,Filter,Nearer to a major town or city,rurality,Smaller rural settlement,,api-only,Personal characteristics
+relative_access,Filter,Nearer to a major town or city,rurality,Urban,,api-only,Personal characteristics
+relative_access,Filter,Total,rurality,Total,,api-only,Personal characteristics
+school_admission_type,Filter,Non-selective schools in highly selective areas,,,,api-only,Establishment characteristics
+school_admission_type,Filter,Non-selective schools in other areas,,,,api-only,Establishment characteristics
+school_admission_type,Filter,Selective schools,,,,api-only,Establishment characteristics
+school_admission_type,Filter,Total,,,,api-only,Establishment characteristics
 school_cohort_size,Filter,1 to 15 pupils,,,,api-only,Establishment characteristics
 school_cohort_size,Filter,16 to 30 pupils,,,,api-only,Establishment characteristics
 school_cohort_size,Filter,31 to 60 pupils,,,,api-only,Establishment characteristics
 school_cohort_size,Filter,61 to 90 pupils,,,,api-only,Establishment characteristics
 school_cohort_size,Filter,91+ pupils,,,,api-only,Establishment characteristics
 school_cohort_size,Filter,Total,,,,api-only,Establishment characteristics
 school_religious_character,Filter,Church of England,,,,api-only,Establishment characteristics
+school_religious_character,Filter,Hindu,,,,api-only,Establishment characteristics
 school_religious_character,Filter,Jewish,,,,api-only,Establishment characteristics
 school_religious_character,Filter,Methodist,,,,api-only,Establishment characteristics
+school_religious_character,Filter,Multi-faith,,,,api-only,Establishment characteristics
 school_religious_character,Filter,Muslim,,,,api-only,Establishment characteristics
 school_religious_character,Filter,No religious character,,,,api-only,Establishment characteristics
 school_religious_character,Filter,Other Christian faith,,,,api-only,Establishment characteristics
 school_religious_character,Filter,Other religious character,,,,api-only,Establishment characteristics
+school_religious_character,Filter,Roman catholic,,,,api-only,Establishment characteristics
 school_religious_character,Filter,Roman Catholic,,,,api-only,Establishment characteristics
```

Comment:
I don't think we should have both versions - quick google suggests it should be capitalised 'Catholic'?